### ISSUE TITLE
EGD: premodern deletion with ui or other vowel marker

### ISSUE THREAD

**Issue by danbalogh** [erc-dharma/project-documentation#88]
Arlo, I'm creating an issue for this topic you raised a while ago so we don't forget about it. No urgency to deal with it, but we should sooner or later.

**Arlo in email:** we need a way to encode deletion using only i, not ui. Suggestion: `sasaṇḍan u<del rend="ui">⌈</del>lun·<!-- only ulu, not suku+ulu, is used to erase the unwanted taling -->` - proposed EGD addition: "and add an XML note in cases where only i or u is found to mark the erasure, as in the example"

**Dan in email:** this is sort of problematic on several points, so we'll need to discuss it.
1, the general idea "add an XML note in cases where only i or u is found to mark the erasure" - this does not sound like a good idea to me. On the one hand, rend="ui" is specifically defined as u and i, so why not use rend="other", defined as " any deletion marker other than those listed above "? If that is because you have certain research questions in mind targeted specifically at the use of "ui" and you don't want a case like this to be excluded from that, then perhaps we should replace rend="ui" with something like rend="extravowel" and define it in a more general way, so that it can cover this case. That said, if this deletion with only "i" instead of "ui" is an anomaly rather than something quite widespread, then I can endorse sticking to rend="ui" in the markup. Sort of analogous to reading a badly formed or incomplete character as the expected character.
On the other hand, but still on this first point: who do you expect will ever read those XML comments? I would much prefer mentioning such cases in the hand description under "other palaeographic observations" (and only there).

2. your `sasaṇḍan u<del rend="ui">⌈</del>lun` is not compatible with the TG, according to which ⌈ is only to be used for prescript components _split off_ from the character to which they belong. If you would like to modify the TG on that count, then let me know and we can consider it, though I would prefer we did not. This whole business involving ⌈ is not very tidy and I would like to keep its use to the absolute minimum where it cannot be avoided. The straightforward way to handle your text sample (unless I misunderstand it, which is possible, as I am not familiar with the script) would be this: `ul<subst><del rend="ui">e</del><add place="inline">u</add></subst>n`. That is to say, the vowel component e was deleted (using an "ui" mark; or whatever rend value we decide on as per my first point), and the vowel u was added to the l.

To sum up:
1. I'd like you to decide on a value of rend to use in such cases; having heard my thoughts on this, you are in the best position to do that. I'll go along with whatever you prefer.
2. I very strongly suggest a mention in the hand description instead of an XML comment.
3. I very strongly suggest encoding in accordance with EGD 4.5.3 and not involving the ⌈ in such a case. Happy to add this as an illustration next to Example 4.5.3.c, once we've decided on `@rend`. If you really prefer ⌈, then I'll need you to suggest exactly how we should rephrase the TG section on ⌈, and in what circumstances we can permit it other than that of split characters.

**Comment by arlogriffiths** [comment]
I don't know of this discussion was ever brough to a conclusion. So am reopening it.

**Comment by danbalogh** [comment]
@arlogriffiths I am unable to reconstruct what this was about. Is it still relevant?

**Comment by danbalogh** [comment]
Having checked on this again, what I can say is that `@rend="other"` is in the EGD for use with any deletion marker other than strikeout and "ui". I believe this must have been introduced precisely for this purpose, and I'm closing the thread.
